main.yml

```diff
@@ -41,12 +41,26 @@ ansible:
   title: Ansible Automation Platform
   frontend:
     sub_apps:
+      - id: ansible-dashboard
       - id: automation-hub
         default: true
```

Comment:
do we want to make the dashboard the default?

Reply:
Not in this PR - after I have more of the dashboard code completed